modules/apps/frontend-js/frontend-js-a11y-web/src/main/resources/META-INF/resources/SDK.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+const PROTOCOL = 'com.liferay.frontend.js.a11y.protocol';
+
+export type ChannelEvent<T, K> = {
+	payload: T;
+	kind: K;
+	protocol: typeof PROTOCOL;
+};
+
+export type Recv<T, S> = (payload: T, kind?: S) => void;
```

Comment:
What means `Recv`?

Reply:
Just a known abbreviation for `receiver`, common in low-level languages, just stealing the name to bring it to the concept of Channel here.

You can find more places that end up using https://www.google.com/search?q=recv&oq=recv+&aqs=chrome..69i57j0i512l2j0i67j0i512l6.2106j0j7&sourceid=chrome&ie=UTF-8, `recv` is also more of a method, as in the channel implementation in Rust https://doc.rust-lang.org/std/sync/mpsc/struct.Receiver.html but here we use it as a callback listener.

Reply:
Usually this is more tied to syscall calls, to receive socket messages for example, so this is a bit similar to Channel too.